### PR TITLE
feat: rename `-Project` scheme to `-Workspace`

### PR DESCRIPTION
**Pull request by danyf90**
Part of splitting https://github.com/tuist/tuist/pull/3987

### CODE REVIEW

**Review by adellibovi — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -933,7 +932,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ### Changed
 
-- Generate multiple `XXX-Project` schemes if there are multiple platforms [#2081](https://github.com/tuist/tuist/pull/2081) by [@fortmarek](https://github.com/fortmarek)
```
Comment (adellibovi) [inline CHANGELOG.md:936]: Maybe let's not rename old reference in the changelog?

**Review by fortmarek — APPROVED** [review]
This is a good name change, it makes it much clearer - I'm a bit surprised we haven't documented this anywhere, maybe it's worth adding an explanation somewhere, wdyt?

File: CHANGELOG.md (hunk 1 above)
Comment (danyf90) [reply CHANGELOG.md:936]: oops

**Comment by danyf90** [comment]
> This is a good name change, it makes it much clearer - I'm a bit surprised we haven't documented this anywhere, maybe it's worth adding an explanation somewhere, wdyt?

Makes sense, but I would do it in a separate PR

**Review by luispadron — APPROVED** [review]
Makes sense since the current logic applies to multiple projects and we always create a workspace.

**Pull request merged**